rotors_hil_interface/include/rotors_hil_interface/hil_interface_node.h

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2016 Pavel Vechersky, ASL, ETH Zurich, Switzerland
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef ROTORS_HIL_INTERFACE_NODE_H_
+#define ROTORS_HIL_INTERFACE_NODE_H_
+
+#include <memory>
+
+#include <rotors_hil_interface/hil_interface.h>
+
+namespace rotors_hil {
+// Default values
+static constexpr bool kDefaultSensorLevelHil = true;
+static constexpr double kDefaultHilFrequency = 100.0;
+static constexpr double kDefaultBodyToSensorsRoll = M_PI;
+static constexpr double kDefaultBodyToSensorsPitch = 0.0;
+static constexpr double kDefaultBodyToSensorsYaw = 0.0;
+static const std::string kDefaultActuatorsPubTopic = "gazebo/command/motor_speed";
+static const std::string kDefaultMavlinkPubTopic = "/mavlink/to";
```

Comment:
`kDefaultActuatorsPubTopic` is no longer present in file. I'm not knowledgeable enough on the MAVlink ones to make a call on this.